pyproject.toml

```diff
@@ -50,3 +50,6 @@
         directory = "misc"
         name = "Misc"
         showcontent = false
+
+[tool.black]
+skip-string-normalization = true
```

Comment:
why?

Reply:
If this is because `u""` is getting changed to `""`, I think that passing `--target-version=py27` might be a more appropriate way to get there.